.github/workflows/build-revive-debian.yml

```diff
@@ -0,0 +1,32 @@
+name: Build revive-debian
+on:
+  workflow_dispatch:
+
+env:
+  REVIVE_DEBIAN_PACKAGE: revive-debian-x86
+  DEBIAN_CONTAINER: revive-builder-debian-x86
+  DEBIAN_CONTAINER_BUILDER: build-debian-builder.sh
+  DEBIAN_CONTAINER_RUNNER: run-debian-builder.sh
+  REVIVE_DEBIAN_INSTALL: ${{ github.workspace }}/target/release
+  REVIVE_DEBIAN_BINARY: resolc
+  RUST_VERSION: stable
```

Comment:
The problem here is `stable` will change over time, but we want to have manual control over it. We can stick to 1.80 (slightly older version should do fine too).

Reply:
Changed to 1.80